app/helpers/application_helper.rb

```diff
@@ -51,13 +51,13 @@ def table_of_contents_separator(options = {})
   end
 
   def manuscript_link(options = {})
-    druid = options[:value]
+    druid = options[:value][0]
     document = options[:document]
     ms_number = document['manuscript_number_tesim']
-    return if druid.blank? || ms_number.blank?
-    return druid if document['format_main_ssim'] != ['Page details']
+    return if druid.blank?
+    return druid if (document['format_main_ssim'] != ['Page details']) || ms_number.blank?
```

Comment:
Should we try to return a link to the druid?

Reply:
Sure, I can give that a swing.